lib/tools/system-calls.js

```diff
@@ -72,8 +72,13 @@ systemCallMethods.getBinaryFromSdkRoot = async function getBinaryFromSdkRoot (bi
   }
 
   const fullBinaryName = this.getBinaryNameForOS(binaryName);
-  const binaryLocs = ['platform-tools', 'emulator', 'tools', `tools${path.sep}bin`]
-    .map((x) => path.resolve(this.sdkRoot, x, fullBinaryName));
+  const binaryLocs = [
+    'platform-tools',
+    'emulator',
+    `cmdline-tools${path.sep}latest${path.sep}bin`,
```

Comment:
I think it makes sense to represent parts of the path as array items, so the resulting assignment might look like

```
const binaryLocs = [
  'platform-tools',
  ...
  ['cmdline-tools', 'latest', 'bin]',
  ...
].map((x) => path.resolve(this.sdkRoot, ...(_.isArray(x) ? x : [x]), fullBinaryName))

```